src/window.rs

```diff
@@ -274,11 +274,14 @@ impl Window {
     pub fn set_cursor_visible(&mut self, visible: bool) {
         if visible != self.cursor_visible {
             self.cursor_visible = visible;
-            self.window.set_cursor_state(if visible {
+            match self.window.set_cursor_state(if visible {
```

Comment:
Since there's not any action to take in the `Ok(())` case, this can be cleaned up slightly by doing

```rust
if let Err(err) = self.window.selt_cursor_state(/* ... */) {

}
```